app/drivers/zephyr/Kconfig

```diff
@@ -13,6 +13,10 @@ config ZMK_KSCAN_DIRECT_POLLING
 	bool "Poll for key event triggers instead of using interrupts on direct wired boards."
     default n
 
+config ZMK_KSCAN_DEMUX_POLLING
```

Comment:
It certainly seems to me, that polling is required given an IO expander like this. From some quick digging, it doesn't seem like there's a way to get this to set *all* the columns to active, in order for the input rows to trigger interrupts for any key press, right?

If that's the case, we way want to look at making the polling rate configurable on this, to be sure users can tweak their power usage accordingly, since it won't be as efficient as using interrupts.

Reply:
That's exactly right. Only one col can be on at once. A psuedo-interrupt configuration would work, whereby the columns are set at a periodic rate and the result isn't read back, but rather pins are configured as interrupts. I'm not sure that it would buy much in the way of power savings so probably wouldn't be worth the effort. I like the idea of configurable poll rate in msec as part of the overlay. I'm going to refactor to remove the interrupt code and implement the configurable polling rate.